lib/ansible/executor/play_iterator.py

```diff
@@ -232,6 +232,12 @@ def __init__(self, inventory, play, play_context, variable_manager, all_vars, st
             # play context to flag that we've started at a task (and future
             # plays won't try to advance)
             play_context.start_at_task = None
+        elif play_context.start_at_task:
```

Comment:
what if the task is on a subsequent play? Won't this error if it is not always in the first play?

Reply:
Ah, yes. Fixed. Thanks!